sema/type.go

```diff
@@ -8043,7 +8043,10 @@ func IsNilType(ty Type) bool {
 	return true
 }
 
+const TransactionTypeName = "transaction"
```

Comment:
This is to be consistent with the static-type of the transaction-value (which gets created by calling the `Value.StaticType()` method)